dalle-image-enhancement/README.md

```diff
@@ -1,17 +1,39 @@
 # DALL-E Candidate Image Editing
-A lot of the input images present in the dataset show partial faces or are images in which the orientation of the figure obstruct the view of the face. Utilizing Dall-E, we were able to modify some candidate images to be able to give a better representation of the image being ran through the model. 
-![](https://github.com/Epicato/lux-AI/blob/main/game.png)
+
+A lot of the input images present in the dataset show partial faces or are images in which the orientation
+of the figure obstruct the view of the face. Utilizing Dall-E, we were able to modify some candidate images
+to be able to give a better representation of the image being ran through the model. 
+
+<!-- 
+Commenting out this image reference since it seems unrelated (TRG)
+![](https://github.com/Epicato/lux-AI/blob/main/game.png) 
+-->
+
 ![alt text](Isolated.png "Title")
```

Comment:
Tariq: There's a reference to an image file 'Isolated.png' which doesn't seem to be included. Can you add?